core/src/main/java/com/backblaze/b2/client/structures/B2UploadProgress.java

```diff
@@ -14,8 +14,35 @@
  *       For example, if we're adding a SHA1 to the thing being uploaded,
  *       the contentLength will be at least 40 bytes longer than you expect
  *       because we're adding an ascii-hex-encoded SHA1.
+ *
+ * NOTE: when storing a large file that involves copying parts, it no
```

Comment:
"it no longer becomes possible"?  "it's not possible"?

Reply:
Done.